src/lib/features/feature-toggle/feature-toggle-store.ts

```diff
@@ -740,21 +740,21 @@ export default class FeatureToggleStore implements IFeatureToggleStore {
                 `LEFT OUTER JOIN ${API_TOKEN_TABLE} AS t on ev.created_by = t.username`,
             )
             .whereRaw(
-                `f.created_by_user_id IS null AND ev.type = 'feature-created'`,
+                `f.created_by_user_id IS null AND
+                ev.type = 'feature-created' AND
+                (u.id IS NOT null OR t.username IS NOT null)`,
```

Comment:
We still need to figure out what to do when we can't resolve the user, we've identified many cases in enterprise and demo instances. 

Any suggestion @chriswk @ivarconr?